### PR TITLE
New path for media

### PR DESCRIPTION
**Pull request by ericgaspar**
- Set media path to `/home/yunohost.app/lufi` cf #51 
 /!\ This is a breaking change cf. https://github.com/YunoHost-Apps/lufi_ynh/blob/c94205d7dd3857ee1edaf2fd5fff73ceed9b3231/conf/lufi.conf.template#L107-L108

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -22,14 +22,22 @@
...
 	"arguments": {
 		"install": [
+			{
+                "name": "warning",
+                "type": "display_text",
+                "ask": {
+                    "en": "/!\\ The media path has been changed to /home/yunohost.app/lufi, the files already uploaded and shared won't be downloadable anymore.",
+                    "fr": "/!\\ Le chemin des médias partagés a été changé en /home/yunohost.app/lufi, les fichiers déjà téléchargés et partagés ne seront plus téléchargeables."
+                }
+            },
```
Comment (yalh76) [inline manifest.json:40]: If it brokes previously uploaded files, it's better not to change the media path

**Review by yalh76 — CHANGES_REQUESTED** [review]
If it brokes previously uploaded files, it's better not to change the media path
And it would be better not to do different things in the same PR: changing the media path and upgrading the version

**Pull request closed without merging**